packages/view/backend/sirius-components-view-tree-edit/src/main/java/org/eclipse/sirius/components/view/tree/provider/FetchTreeItemContextMenuEntryItemProvider.java

```diff
@@ -25,17 +25,15 @@
 import org.eclipse.sirius.components.view.tree.TreePackage;
 
 /**
- * This is the item provider adapter for a
- * {@link org.eclipse.sirius.components.view.tree.FetchTreeItemContextMenuEntry}
+ * This is the item provider adapter for a {@link org.eclipse.sirius.components.view.tree.FetchTreeItemContextMenuEntry}
  * object. <!-- begin-user-doc --> <!-- end-user-doc -->
  *
  * @generated
  */
 public class FetchTreeItemContextMenuEntryItemProvider extends TreeItemContextMenuEntryItemProvider {
 
     /**
-     * This constructs an instance from a factory and a notifier. <!--
-     * begin-user-doc --> <!-- end-user-doc -->
+     * This constructs an instance from a factory and a notifier. <!-- begin-user-doc --> <!-- end-user-doc -->
```

Comment:
I tried to fix the formatting but couldn't make it work. I guess my configuration is different from the one of the previous committer. Let me know if this is not acceptable and I'll manually update it.